### PR TITLE
add to parachain backing pallet

### PR DESCRIPTION
**Pull request by xiaoch05**

### CODE REVIEW

File: runtime/crab/src/pallets/to_parachain_backing_.rs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	fn outbound_latest_generated_nonce(lane_id: LaneId) -> u64 {
+		BridgeCrabParachainMessages::outbound_latest_generated_nonce(lane_id).into()
+	}
+
+	fn inbound_latest_received_nonce(lane_id: LaneId) -> u64 {
+		BridgeCrabParachainMessages::inbound_latest_received_nonce(lane_id).into()
+	}
+}
+
+pub const CRAB_PARACHAIN_ISSUING_PALLET_INDEX: u8 = 24;
```
Comment (AurevoirXavier) [inline runtime/crab/src/pallets/to_parachain_backing_.rs:25]: Suggest moving the const vars to the top of this file.

File: runtime/crab/src/pallets/to_parachain_backing_.rs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			call,
+			dispatch_fee_payment,
+		})
+	}
+}
+
+frame_support::parameter_types! {
+	pub const CrabParachainChainId: ChainId = CRAB_PARACHAIN_ID;
+	pub const S2sBackingPalletId: PalletId = PalletId(*b"pl/s2sba");
+	pub const MaxLockRingAmountPerTx: Balance = 10_000 * COIN;
```
Comment (HackFisher) [inline runtime/crab/src/pallets/to_parachain_backing_.rs:65]:
```suggestion
	pub const MaxLockRingAmountPerTx: Balance = 1_000_000 * COIN;
```

**Review by HackFisher — COMMENTED** [review]
@xiaoch05 There are conflicts

**Comment by AurevoirXavier** [comment]
I think there are some issues in this PR.
And I pushed some fixes in [`fc7823f` (#914)](https://github.com/darwinia-network/darwinia/pull/914/commits/fc7823f65d3dd8c1d7581b8227e4a28822d7ab1f).

**Pull request merged**